hydra-node/src/Hydra/Node.hs

```diff
@@ -68,15 +69,20 @@ data HydraNode tx m = HydraNode
   , env :: Environment
   }
 
+-- NOTE(AB): we use partial fields access here for convenience purpose, to
+-- make serialisation To/From JSON straightforward
 data HydraNodeLog tx
-  = ErrorHandlingEvent Party (Event tx) (LogicError tx)
-  | ProcessingEvent Party (Event tx)
-  | ProcessedEvent Party (Event tx)
-  | ProcessingEffect Party (Effect tx)
-  | ProcessedEffect Party (Effect tx)
+  = ErrorHandlingEvent {emitter :: Party, event :: Event tx, reason :: LogicError tx}
+  | ProcessingEvent {emitter :: Party, event :: Event tx}
+  | ProcessedEvent {emitter :: Party, event :: Event tx}
+  | ProcessingEffect {emitter :: Party, effect :: Effect tx}
+  | ProcessedEffect {emitter :: Party, effect :: Effect tx}
```

Comment:
`emitter` is a bit ambiguous to me. I think I would (mis-)understand it as which party created an event or so. In fact, these are logs / traces and it is the party which was emitting the trace, right? Maybe `traceEmitter` or `loggedBy` or something else would fit better?

Reply:
how about simply `party` ?

Reply:
Okay, although there are other `party` fields "further down". So it is not adding to clarity much, but it's slightly less ambiguous